lib/PgSQL_Thread.cpp

```
@@ -4417,6 +4424,27 @@ SQLite3_result* PgSQL_Threads_Handler::SQL3_GlobalStatus(bool _memory) {
+		result->add_row(pta);
+	}
+	{	// Poisoned sessions recovered via client ROLLBACK / COMMIT / ABORT.
+		pta[0] = (char*)"pgsql_tx_poisoned_recovered_total";
+		sprintf(buf, "%llu", get_tx_poisoned_recovered_total());
+		pta[1] = buf;
+		result->add_row(pta);
+	}
+	{	// Client statements rejected with ERROR 25P02 while the session was
+		// in the poisoned state (includes extended-query P/B/D/C/E/S while
+		// poisoned, and any non-recovery simple-query statement).
+		pta[0] = (char*)"pgsql_tx_poisoned_rejected_statements_total";
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)